### DISCUSSION TITLE
Can not get the Tile working properly

### DISCUSSION THREAD

**Discussion by dimitrove** [fltk-rs/fltk-rs#823] [Q&A] [answered]
The border between left Pack and right Pack kind of moves, but the shifted image is painted over the old image...
`
use fltk::{prelude::*, *};

fn main() {
    let app = app::App::default();
    let mut wind = window::Window::new(100, 100, 620, 320, "FLTK");

    // outer HPack
    let hpack = group::Pack::default()
        .with_type(group::PackType::Horizontal).with_size(604, 300);

    // SplitPane
    let split = group::Tile::default().with_size(600, 300);

    // VPack left
    let mut vpack_left = group::Pack::default()
        .with_type(group::PackType::Vertical).with_size(300, 300).with_pos(0, 0);
    vpack_left.set_frame(enums::FrameType::UpFrame);
    frame::Frame::default().with_size(0, 60).with_label("left");
    vpack_left.end();

    // VPack right
    let mut vpack_right = group::Pack::default()
        .with_type(group::PackType::Vertical).with_size(300, 300).right_of(&vpack_left, 4);
    vpack_right.set_frame(enums::FrameType::UpFrame);
    frame::Frame::default().with_size(0, 60).with_label("right");
    vpack_right.end();

    split.end();

    hpack.end();

    wind.end();
    wind.show();

    // !!!
    app.run().unwrap();
}
`

**Comment by MoAlyousef** [comment, marked as answer]
The drawing artifact is a known issue in FLTK:
https://github.com/fltk/fltk/issues/263

Regarding tiles, the FLTK docs state that all children elements should be exactly adjacent:
```rust
use fltk::{prelude::*, *};

fn main() {
let app = app::App::default();
let mut wind = window::Window::new(100, 100, 620, 320, "FLTK");


// outer HPack
let hpack = group::Pack::default()
    .with_type(group::PackType::Horizontal).with_size(600, 300);

// SplitPane
let mut split = group::Tile::default().with_size(600, 300);

// VPack left
let mut vpack_left = group::Pack::default()
    .with_type(group::PackType::Vertical).with_size(300, 300).with_pos(0, 0);
vpack_left.set_frame(enums::FrameType::UpFrame);
let mut frm = frame::Frame::default().with_size(0, 60).with_label("left");
frm.set_frame(enums::FrameType::FlatBox); // add a flatbox to avoid NoBox artifacts
vpack_left.end();

// VPack right
let mut vpack_right = group::Pack::default()
    .with_type(group::PackType::Vertical).with_size(300, 300).right_of(&vpack_left, 0); // changed padding to 0
vpack_right.set_frame(enums::FrameType::UpFrame);
let mut frm = frame::Frame::default().with_size(0, 60).with_label("right");
frm.set_frame(enums::FrameType::FlatBox); // add a flatbox to avoid NoBox artifacts
vpack_right.end();

split.end();

hpack.end();

wind.end();
wind.show();

// !!!
app.run().unwrap();
}
```
